spec/Gist_Attributes_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../Gist_Attributes.rb'
+require 'spec_helper'
+
+RSpec.describe GistAttributes do
+  describe "File_exist" do
+    it "Archivo existente" do
```

Comment:
Be aware that rspec checks if code behaves how it is supposed to. Example:
```
RSpec.describe "something" do
  it "does something" do
  end
end
```

So in this case, you could write:
```
describe "exist?" do
    it "should be true if file exists" do
        ...
    end
end
```

And within the test you should call the function `exist?`